doc/releases/changelog-0.20.0.md

```diff
@@ -1,82 +1,121 @@
 :orphan:
 
-# Release 0.20.0-dev (development release)
+# Release 0.20.0 (current release)
 
 <h3>New features since last release</h3>
 
-* Added functions for computing the values of atomic and molecular orbitals at a given position.
-  [(#1867)](https://github.com/PennyLaneAI/pennylane/pull/1867)
+<h4>Shiny new circuit drawer!🎨🖌️ </h4>
 
-  The functions `atomic_orbital` and `molecular_orbital` can be used, as shown in the
-  following codeblock, to evaluate the orbitals. By generating values of the orbitals at different
-  positions, one can plot the spatial shape of a desired orbital.
+* PennyLane now supports drawing a QNode with matplotlib!
+  [(#1803)](https://github.com/PennyLaneAI/pennylane/pull/1803)
 
   ```python
-  symbols  = ['H', 'H']
-  geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.0]], requires_grad = False) 
-  mol = hf.Molecule(symbols, geometry)
-  hf.generate_scf(mol)()
+  dev = qml.device("default.qubit", wires=4)
 
-  ao = mol.atomic_orbital(0)
-  mo = mol.molecular_orbital(1)
+  @qml.qnode(dev)
+  def circuit(x, z):
+      qml.QFT(wires=(0,1,2,3))
+      qml.Toffoli(wires=(0,1,2))
+      qml.CSWAP(wires=(0,2,3))
+      qml.RX(x, wires=0)
+      qml.CRZ(z, wires=(3,0))
+      return qml.expval(qml.PauliZ(0))
+  fig, ax = qml.draw_mpl(circuit)(1.2345, 1.2345)
+  fig.show()
   ```
 
-  ```pycon
-  >>> print(ao(0.0, 0.0, 0.0))
-  >>> print(mo(0.0, 0.0, 0.0))
-  0.6282468778183719
-  0.018251285973461928
-  ```
+  <img src="https://pennylane.readthedocs.io/en/latest/_static/draw_mpl_qnode/main_example.png" width=70%/>
```

Comment:
@albi3ro this image doesn't seem to be available anymore, how come?

Reply:
```suggestion
  <img src="https://pennylane.readthedocs.io/en/latest/_images/main_example.png" width=70%/>
```
Found the proper link.

Reply:
Thanks!